core-war/src/main/webapp/attachment/jsp/editAttachedFiles.jsp

```diff
@@ -38,13 +38,16 @@
 <%@ page import="org.silverpeas.core.contribution.attachment.model.SimpleDocument" %>
 <%@ page import="org.silverpeas.web.attachment.VersioningSessionController" %>
 <%@ page import="org.silverpeas.core.ResourceReference" %>
+<%@ page import="org.silverpeas.core.contribution.attachment.process.huge.AttachmentHugeProcessManager" %>
 <%@ include file="checkAttachment.jsp"%>
 <view:setConstant var="spinfire" constant="org.silverpeas.core.util.MimeTypes.SPINFIRE_MIME_TYPE" />
 <c:set var="mainSessionController" value="<%=m_MainSessionCtrl%>" />
 <view:settings var="onlineEditingEnable" settings="org.silverpeas.util.attachment.Attachment" defaultValue="${false}" key="OnlineEditingEnable" />
 <view:settings var="dAndDropEnable" settings="org.silverpeas.util.attachment.Attachment" defaultValue="${false}" key="DragAndDropEnable" />
 <c:set var="webdavEditingEnable" value="${mainSessionController.webDAVEditingEnabled && onlineEditingEnable}" />
```

Comment:
Shouldn't the WebDAV editing be disabled if a huge processing is currently done on attachments?

Reply:
I suppose it is disabled as it is explained in the previous comment

Reply:
Yes, but I have updated the source anyway because of some missings.